lessons/fundamentals/connect-with-rpc.mdx

```diff
@@ -42,17 +42,19 @@ tutorial or other source.
 In blockchain development, it is necessary to connect your dApp to the
 blockchain to deploy smart contracts and interact with the deployed contract.
 This can be done by either running your own blockchain node such as GETH or
-accessing the blockchain through an RPC endpoint service such as Infura, Alchemy
-or Ankr. Connecting through the 3 providers is much easier than running your own
-node. With all 3 services, it is quick, easy and **FREE** to open an account and
-get access to RPC endpoints of many different testnets including Goerli
+accessing the blockchain through an RPC endpoint service such as Infura, Alchemy, 
+Chainstack, or Ankr. Connecting through the 4 providers is much easier than running 
```

Comment:
Connecting through these providers is much easier than running your own node. With each service, it is ....

Reply:
Great suggestion, I've now removed the numbers and changed the language around to accommodate